from ..exceptions import ClientError


class AccountAlreadyExist(ClientError):
    pass
